libs/barista-components/overlay/src/overlay-trigger.ts

```diff
@@ -93,16 +89,13 @@ export class DtOverlayTrigger<T> extends _DtOverlayTriggerMixin
     private _ngZone: NgZone,
     private _focusMonitor: FocusMonitor,
     @Attribute('tabindex') tabIndex: string,
+    // @ts-ignore
```

Comment:
May you explain why typescript failed here on this line?

Or can we remove that?

Reply:
Oh good catch - this was a mistake! 👍

Reply:
Or not :D it is also unused. :D also did not check the file I am in while reading the comment. 

So it is unused since we don't need the platform anymore.